spec/helpers/visit_helper_spec.rb

```diff
@@ -60,6 +60,17 @@ def visit
       expect(helper.prison_address).to start_with "1 Fort Road"
     end
 
+    context "with illegal html in the address" do
+      before :each do
+        expect(Rails.configuration.prison_data['Rochester']).to receive(:[]).
+        with('address') { ['Danger<script>ous'] }
+      end
+
+      it "doesn't return illegal html" do
+        expect(helper.prison_address).to eq("Danger")
```

Comment:
This is quite tightly coupled to the exact behaviour of `sanitize`. It might be better to test that it *doesn't* contain `<`.